support/android/apk/app/build.gradle

```diff
@@ -0,0 +1,302 @@
+apply plugin: 'com.android.application'
+
+
+import groovy.io.FileType
+import org.apache.tools.ant.taskdefs.condition.Os
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
+
+    defaultConfig {
+        applicationId "com.mozilla.servo"
+        minSdkVersion 18
```

Comment:
Older devices are still widely used (think developing countries and regional differences), and several of our test devices also run Android 4.4. I'd like API 18 support too.